### DISCUSSION TITLE
Save files on google drive

### DISCUSSION THREAD

**Discussion by SamuLng** [xournalpp/xournalpp#3096] [Q&A]
I would like to save my files on google drive but i can't find my google drive folder when i do "save as". Is there a way or do I have to save them locally?

**Comment by Technius** [comment]
What operating system are you using?

You can save your files locally and upload them to Google Drive manually (e.g. through the web interface), but there are probably better ways.

**Reply by SamuLng** [reply]
I'm using Pop os. In my file manager i have the google drive folder, i usually upload files from there without using the web interface

**Reply by Technius** [reply]
Unfortunately, Xournal++ currently doesn't support network drives (I'm not sure what the exact limitation is). See #1269 for some potential workarounds.

**Reply by SamuLng** [reply]
Thanks

**Comment by personalizedrefrigerator** [comment]
On Fedora (it's probably similar on PoP OS).

1. Open the file manager
    * ![Click on the Google drive account in the sidebar](https://user-images.githubusercontent.com/46334387/117523124-f0212e80-af6b-11eb-8bcc-da30bb8635c2.png)
    * It should now have an ![eject icon](https://user-images.githubusercontent.com/46334387/117522927-dfbc8400-af6a-11eb-819e-caa899433b99.png) next to it.

2. In Xournal++, `File > Save` As a document.
3. In the save dialog that appears, select "Other Locations"
![Other Locations > Username](https://user-images.githubusercontent.com/46334387/117523215-6faefd80-af6c-11eb-8ec8-20e29dd0b761.png)
4. Under `networks`, the drive should be visible. Click on it and save there.

**Reply by Technius** [reply]
I think this is dependent on the OS. On Solus, I don't have options to access network drives in the file chooser.

**Reply by SamuLng** [reply]
thanks, i found it. I guess it's some pop bug because I had checked several times but found nothing under network

**Reply by personalizedrefrigerator** [reply]
Step 1 (mounting the network drive) is really important. Could it be that (and not a PoP bug)?

**Reply by SamuLng** [reply]
yes I think that's the problem, I thought it did it automatically at startup
